src/registrar/templates/emails/update_to_approved_domain.txt

```diff
@@ -0,0 +1,26 @@
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
+
+Hi,
+An update was made to a domain you manage.
+DOMAIN: {{domain}}
+UPDATED BY: {{user}}
+UPDATED ON: {{date}}
+INFORMATION UPDATED: {{changes}}
```

Comment:
For the field that displays “**Information updated,**” can you make the following adjustments:
  - Nameservers should be two words: **Name servers**
  - Org Name/Mailing should be changed to: **Organization details**
  - Senior Official should use lowercase for second word: **Senior official**
  - Security Email should use lowercase for second word: **Security email**
  - DS Data should be: **DNSSEC / DS Data**

Reply:
Can do!